analytics/analytics_package/analytics/sheets_api.py

```diff
@@ -0,0 +1,244 @@
+import gspread
+import gspread_formatting
+from enum import Enum
+from googleapiclient.discovery import build
+
+class FILE_OVERRIDE_BEHAVIORS(Enum):
+    OVERRIDE_IF_IN_SAME_PLACE = 1
+    EXIT_IF_IN_SAME_PLACE = 2
+    EXIT_ANYWHERE = 3
+
+class WORKSHEET_OVERRIDE_BEHAVIORS(Enum):
+    OVERRIDE = 1
+    EXIT = 2
+
+FONT_SIZE_PTS = 10
+PTS_PIXELS_RATIO = 4/3
+DEFAULT_BUFFER_CHARS = 2
+
+def extract_credentials(authentication_response):
+    """Extracts the credentials from the tuple from api.authenticate"""
+    return authentication_response[3]
+
+def authenticate_gspread(authentication_response):
+    """Authenticates the gspread client using the credentials in the tuple from api.authenticate"""
+    gc = gspread.authorize(extract_credentials(authentication_response))
+    return gc
+
+def authenticate_drive_api(authentication_response):
+    """Authenticates the Drive API using the credentials in the tuple from api.authenticate"""
+    return build('drive', 'v3', credentials=extract_credentials(authentication_response))
```

Comment:
The tuple returned from calling `api.authenticate` with `drive_service_params` should have the Drive API as its first item, so I don't think calling `build` here is necessary